Fix build failure with latest autotools which now generates Makefiles
that build programs before libraries instead of after.
--- Makefile.am.orig	2016-07-19 05:42:18.000000000 -0500
+++ Makefile.am	2018-08-26 00:56:22.000000000 -0500
@@ -10,4 +10,4 @@
 libsphinxclient_la_LDFLAGS = -release @VERSION@
 
 include_HEADERS =   sphinxclient.h 
-test_LDADD = .libs/libsphinxclient.a
+test_LDADD = libsphinxclient.la
